facia-press/app/frontpress/FapiFrontPress.scala

```diff
@@ -183,12 +184,41 @@ trait FapiFrontPress extends Logging with ExecutionContexts {
       .map(_.map(PressedContent.make))
   }
 
+  private def isHighPriorityCollection(collection: CollectionConfigJson): Boolean = {
+    collection.metadata.exists(_.exists({
+      case Special | Canonical => true
```

Comment:
I know I really shouldn't be looking at this while on holiday, but I had to open my work email to look for a holiday-related email, so... here I am. Just to say this is the relevant code in the old email renderer, which may be helpful:

- https://github.com/guardian/gu-email-renderer/blob/a91bcf5da07ba4bb511d1e20d007b0e6697a030a/template/us/daily/categories_us.html#L19-L63
- https://github.com/guardian/gu-email-renderer/blob/422446b3c79e43fefe03407f93e1d8bffee5663d/template/uk/daily/categories.html#L19-L43
- https://github.com/guardian/gu-email-renderer/blob/422446b3c79e43fefe03407f93e1d8bffee5663d/template/au/daily/v4.html#L19-L43